scripts/hash-exec-copy.py

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python3
+
+import argparse
+from datetime import datetime
+import requests
+import subprocess
+
+INPUT_DATE_FORMAT="%Y-%m-%d"
+REPO_URL = "/makerdao/community"
+
+def get_executive_by_title (exec_title):
+    base_git_api_url = f"https://api.github.com/repos{REPO_URL}/commits"
+
+    api_response = requests.get(
+        f"{base_git_api_url}?path=governance/votes/{exec_title}",
+        params = {'per_page': '1'}
+    )
+    api_response.raise_for_status()
+    commits = api_response.json()
+    if not commits:
+        raise ValueError(f"Executive copy not found: {exec_title}")
+
+    commit_hash = commits[0].get("sha", "")
+
+    # Get target exec copy content
+    exec_copy_response = requests.get(
+        f"https://raw.githubusercontent.com{REPO_URL}/{commit_hash}/governance/votes/{exec_title}"
+    )
+    exec_copy_response.raise_for_status()
+
+    # Get target exec copy content
+    executive_url = exec_copy_response.url
+
+    # Remove trailing whitespace as command substitution in shell script removes it and we want to keep the output consistent
+    # Example of shell script to create hash `cast keccak -- "$(wget 'PATH' -q -O - 2>/dev/null)"`
+    if exec_copy_response.text[-1] == '\n':
+        content = exec_copy_response.text[:-1]
+
+    return content, executive_url, commit_hash
+
+def get_content_hash (content):
+    try:
+        # Run the 'cast keccak' command with the content
+        keccak_result = subprocess.run(
+            ['cast', 'keccak', '--', content],  # Run the cast keccak command
+            stdout = subprocess.PIPE,  # Capture the output
+            stderr = subprocess.PIPE,  # Capture the errors
+            text = True,  # Return output as text, not bytes
+            check = True  # Raise an exception if the command fails
+        )
```

Comment:
To make this truly platform-independent we need to remove this subprocess call and replace it with python-native keccak algorithm.

But I think this can be done in a subsequent PR – rewriting everything else into python already improves it significant enough to catch the problem we encountered in the previous spell